lib/parser.dart

```diff
@@ -179,6 +179,21 @@ final _legacyPseudoElements = Set<String>.from(const [
   'first-line',
 ]);
 
+// CSS3 functional pseudo-classes (https://www.w3.org/TR/2018/REC-selectors-3-20181106/#selectors)
```

Comment:
Maybe bump the min SDK to 2.2 and use an actual `Set` constant?

Reply:
Fixed